contracts/feature-tests/abi-tester/tests/abi_tester_abi_test.rs

```diff
@@ -82,3 +85,28 @@ fn check_multi_contract_config() {
         vec!["external_view", "payable_any_token", "label_a"]
     );
 }
+
+#[test]
+fn abi_deserialization_check() {
+    let main_json = fs::read_to_string("./abi_tester_expected_main.abi.json").unwrap();
+    let main_abi = multiversx_sc_meta::abi_json::deserialize_abi_from_json(&main_json).unwrap();
+    let abi_enum_type = main_abi
+        .types
+        .get("AbiEnum")
+        .unwrap()
+        .to_type_description("AbiEnum");
+    if let TypeContents::Enum(variants) = abi_enum_type.contents {
+        assert_eq!(variants.len(), 4);
+        assert_eq!(
+            variants[0],
+            EnumVariantDescription {
```

Comment:
make default?

Reply:
It's not default, it's a specific one from the ABI

Reply:
I might make a builder pattern around it, but I wanted to keep this PR quick.